Content.Client/Wires/UI/WiresMenu.cs

```diff
@@ -584,17 +584,12 @@ public StatusLight(StatusLightData data, IResourceCache resourceCache)
 
         private sealed class HelpPopup : Popup
         {
-            private const string Text = "Click on the gold contacts with a multitool in hand to pulse their wire.\n" +
-                                        "Click on the wires with a pair of wirecutters in hand to cut/mend them.\n\n" +
-                                        "The lights at the top show the state of the machine, " +
-                                        "messing with wires will probably do stuff to them.\n" +
-                                        "Wire layouts are different each round, " +
-                                        "but consistent between machines of the same type.";
+            private readonly string _text = Loc.GetString("wires-menu-help-popup");
```

Comment:
No need to make this an instance variable, just resolve it in the function that uses it.

Reply:
Done!